data_access_control.md

```diff
@@ -0,0 +1,117 @@
+Log in to the dashboard as Forecaster A and follow the Sites link to view the list of sites available to you.
+Here you should see 'Power Plant X' provided by Utility X and 'Ashland OR' provided by Organization 1.
+
+Click on 'Power Plant X'. Here you'll find the irradiance and power observations provided by Utility X. Notice
+that the only visible forecast objects are provided by Forecast Provider A.
+
+Return to the site listing and click on 'Ashland OR'. You will find irradiance observations provided by Organization1
+and a day ahead GHI forecast from Forecaster Provider A.
+
+Logging in as Forecaster B should yield all of the same observation and site information, but only forecasts from
+Forecast Provider B should be visible.
+
+## View the roles and permissions of Forecaster A
+Access the User Administration page through the 'Account' menu as in the Utility X permission section above. This section
+should be the same for Forecaster A or B, save for roles allowing each Forecaster to access their own data.
+
+### The User Page
+Click on the User ID field of the users table to view Forecaster A's user information. Notice the 'Read Plant X' role from
+organization Utility X. This role grants the forecaster access to Power Plant X and the observation data associated
+with it. There is also a 'Forecaster A' role that allows the user access to data in its own organization.
+
+### The Role Page
+Click on the 'Read Plant X' role. On this role page you can see the permissions granted to Forecaster A by Utility x. In this
+case the utility has created a single role to apply to both forecasters. Notice the following read permissions:
+  - The 'Read Plant X observation metadata' permission grants the forecaster permission to access metadata about an
+observation. The metadata of an observation is defined in the
+[Observations section of the Data Model](https://solarforecastarbiter.org/datamodel/#observations) document.
+
+  - The 'Read Plant X observation values' permission allows reading values of the observations made by Utility X at Power
+Plant X.
+
+  - The 'Read Plant X site metadata' permission allows the forecaster to read the site's metadata. Site metadata is defined
+in the [Site section of the Data Model](https://solarforecastarbiter.org/datamodel/#site) document.
+
+  - Again, the other permissions allow the Forecast user to view the role and permissions, as these permissions are usually
+reserved for administrators.
+
+### The Permission Page
+Click on the 'Read Plant X observation values' permission. The permission page shows some information about the permission
+as well as listing the objects that it allows its action on. In this case, all of the observations made by Utility X at Power
+Plant X are listed. If Utility X no longer wanted to share data from a portion of these observations, they would simply need
+to remove those observations from the permissions, although editting roles and permissions through the dashboard is not yet
```

Comment:
to remove those observations from the permissions. Support for editing roles and permissions through the dashboard will be implemented soon.